packages/playground/src/components/openapi-output.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent, useCallback, useState } from "react";
+import { OutputEditor } from "./cadl-editor";
+import { PlaygroundManifest } from "../manifest";
+import { SwaggerUI } from "./swagger-ui";
+export interface OpenAPIOutputProps {
+  content: string;
+}
+
+export const OpenAPIOutput: FunctionComponent<OpenAPIOutputProps> = (props)=> {
+  const[selected, setSelected] = useState<"raw"|"swagger-ui">("raw");
+  const handleSelected = useCallback(
+    () => { setSelected(selected === "raw"?("swagger-ui"):("raw"));
+    },
+    [selected],
+  );
+  return(
+    <>
+    {PlaygroundManifest.enableSwaggerUI? (
+    <select className="output-dropdown" onChange={handleSelected} value={selected}>
```

Comment:
seems to be something broken here the dropdown value doesn't get updated in the UI https://cadlplayground.z22.web.core.windows.net/prs/716/